server/test/filters/RecordCookieSizeFilterTest.java

```diff
@@ -15,17 +17,58 @@
 import play.mvc.Http;
 import play.mvc.Result;
 import play.mvc.Results;
+import services.settings.SettingDescription;
+import services.settings.SettingMode;
+import services.settings.SettingType;
+import services.settings.SettingsManifest;
+import services.settings.SettingsSection;
 
 public class RecordCookieSizeFilterTest {
   @Before
   public void clearRegistry() {
     CollectorRegistry.defaultRegistry.clear();
   }
 
+  private SettingsManifest createSettingsManifest(boolean metricsEnabled) {
```

Comment:
Would a mock for settings manifest work instead? 

SettingsManifest mockSettingsManifest = Mockito.mock(SettingsManifest.class);

And then in the test, you could do:
    when(mockSettingsManifest.getCiviformServerMetricsEnabled())
        .thenReturn(true);

Reply:
Thanks for this suggestion - much cleaner!